### DISCUSSION TITLE
Make CP front end link reflect "requestedSite.baseUrl"

### DISCUSSION THREAD

**Discussion by ZapaNOR** [craftcms/cms#12402] [Ideas]
A small annoyance I have had when editing in Craft CMS is that whenever I'm working on a multi site, clicking the company logo / name takes me to the primary site's front end.

![Screenshot 2022-11-30 at 16 06 58](https://user-images.githubusercontent.com/53942798/204834030-307551c7-63b6-473c-8fe6-f2ba34cfc1f2.png)

Updating craftcms/cms/src/templates/_layouts/components/global-sidebar.twig to use requestedSite.baseUrl isntead of siteUrl and requestedSite.name instead of systemName would solve this.

**Comment by brandonkelly** [comment]
I’m not sure about this, since the system name doesn’t change when you switch to a different site within the current page’s toolbar. Maybe we should consider replacing the “system name” with the selected site name, and have it double as the site selector. We’ve considered doing that in the past and it felt a little awkward since not everything within the control panel is actually site-specific. We can maybe reconsider though.

**Reply by ZapaNOR** [reply]
I think some kind of global site selector / notice would be nice. Like you said, not everything is site specific, but right now its a bit messy keeping track of what site you are actually on. 

For example, if you are editing an entry in a section that saves only to current site, there is no indication what site you are actually on when editing / creating that entry. You need to go back to "Entries" to see what site is chosen in the site selector.

My wish would be more emphasis on what site you are currently "on", and rather highlight changes / settings etc. that can affect other sites, not the other way around. Our content creators have to pop into a section or globals to check what site is currently active.

Maybe simply moving the site selector to the top bar would solve most of it?

![Screenshot 2022-12-13 at 09 11 26](https://user-images.githubusercontent.com/53942798/207262280-2c06fb7f-cc4a-4e6a-82bc-7fca78fc7a54.png)

Now Im going off topic, but when talking about the site selector, hiding the sites that are disabled would be appreciated or adding another setting on sites that hides them. Some of our "basic site" configs have 3-4 pre-made sites with all the settings configured in .env, most sites only uses 1, but all of them will show "Site 2, Site 3, Site 4" in the site selector, even tho they are disabled.
